bob-backend/src/pearl/group.rs

```diff
@@ -153,7 +153,9 @@ impl Group {
             .collect();
```

Comment:
I think, we should go further and optimize away this vector creation. `holders` can be iterated with regular `for` loop and during this iteration we can find easily the one that fits into interval (`gets_into_interval(...)` is true) and has maximum start timestamp (`start_timestamp()`)